serialization/physics.proto

```diff
@@ -97,13 +97,16 @@ message DiscreteTrajectory {
     required Point instant = 1;
     required Pair degrees_of_freedom = 2;
   }
-  message Litter {
+  message Child {
     required Point fork_time = 1;
     repeated DiscreteTrajectory trajectories = 2;
```

Comment:
If this message is called Child this field cannot be repeated; revert this renaming.

Reply:
Brood.